src/js/utils.js

```diff
@@ -16,13 +16,24 @@ exports.parseQueryParams = () => {
 
 /** Creates an object based on a form's current input values. */
 exports.parseForm = form => {
-  const formData = new FormData(form);
```

Comment:
I'm just thinking, if we have to have two implementations then maybe we should _just_ use the non-FormData one?

Reply:
Agree, that makes it more maintainable.